schema.py

```diff
@@ -0,0 +1,118 @@
+from sqlalchemy import (Column, ForeignKey, DateTime, Boolean, Integer, Float, String)
+from sqlalchemy.ext.declarative import declarative_base
+from sqlalchemy.orm import relationship, sessionmaker
+from sqlalchemy import create_engine, select
+from datetime import datetime
+import db
+
+
+"""
+This is the database for the webapp ReeDistribute
+All information collected by this program defaults to anonymous 
+
+"""
+Base = declarative_base() #Make it an OOP
+
+class IdPrimaryKeyMixin(object):
+    #An ID column we wish to add to all rows, it will be unique to the table and given incrementaly
+    id = Column(Integer, primary_key=True, nullable = False)
+
+
+class DateTimeMixin(object):
+    #To Record the time each object was created and last updated
+    created_on = Column(DateTime, default=datetime.now)
+    updated_on = Column(DateTime, default=datetime.now, onupdate=datetime.now)
+
+
+class Account(Base, IdPrimaryKeyMixin, DateTimeMixin):
+    """
+    accounts is the table for the Account objects which hold the user information 
+    """
+    __tablename__='accounts' #Store all account objects in accounts
+
+    # Columns:
+    username = Column(String, nullable = False) #used for personalization

```

Comment:
Style, inverse of above: in constructors and function calls, _no space_ on either side of the `=`, eg:

```
Column(String, nullable=False)
```